### PR TITLE
Updates verify message

### LINKED ISSUE #164: Language for email verify notification
via @designlanguage 
User tests revealed that people wanted to know how much time they had to complete registration by email verification. Here's a thought for how we can update that language:

screen shot 2017-09-18 at 10 02 22 am

Verify your email to activate your account. 

An email with a verification code has been sent to george@bank0.com. 
The code will expire in 10 minutes. If you would like a new code, or you haven’t received the email, click here to send a new one.

Thoughts @awolfe76 @keileyo @wpears

### PR DESCRIPTION
**Pull request by wpears**
Depends on #166 
Closes #164 

screen shot 2017-09-21 at 12 41 41 am

### CODE REVIEW

File: keycloak/themes/hmda/login/template.ftl [hunk 1]
```diff
@@ -98,10 +98,15 @@
     </div>
   </#if>
 
-  <div class="usa-grid usa-grid-small">
+  <#assign className="usa-grid usa-grid-small"/>
+  <#if (displayMessage && message?has_content && message.summary == "verifyEmailMessage")>
```
Comment (awolfe76) [inline keycloak/themes/hmda/login/template.ftl]: Just from my overall comment ... is there a reason to make this message different from others here?

Comment (wpears) [reply keycloak/themes/hmda/login/template.ftl]: The reason this is separated here is to separate out the help text (which is automatically inserted in the body of all other error messages, whereas, per #164, the help text was out of the error box.

Comment (awolfe76) [reply keycloak/themes/hmda/login/template.ftl]: 👍

File: keycloak/themes/hmda/login/login-verify-email.ftl [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        ${msg("emailVerifyTitle")}
+    <#elseif section = "header">
+        ${msg("emailVerifyTitle")}
+    <#elseif section = "form">
+      <div class="usa-width-one-whole margin-bottom-1">
+        <div class="usa-alert usa-alert-${message.type}">
+          <div class="usa-alert-body">
+            <h3 style="margin-top: 0;">Verify your email to activate your account</h3>
+            <p>You have been sent an email with a verification code.</p>
+            <p>This code will expire in 10 minutes. If you would like a new code or if you haven't received the email, <strong><a style="text-transform: lowercase;" href="${url.loginAction}">${msg("doClickHere")}</a></strong> to send a new one.</p>
```
Comment (awolfe76) [inline keycloak/themes/hmda/login/login-verify-email.ftl]: I think this should be `60 minutes`.

From the email `This link will expire within 60 minutes.`.

**Pull request merged**